src/layer.ts

```diff
@@ -18,12 +18,11 @@ export class FunctionalTileLayer extends L.TileLayer {
     options?: L.TileLayerOptions
   ) {
     const defaultOptions: L.TileLayerOptions = {
-      tileSize: 384,
+      tileSize: L.Browser.retina ? 384 / 2 : 384,
```

Comment:
Should use `getTileSize()` here as well to get DRY code.

Reply:
Fixed.